app/test/tool/backfill/backfill_packageversions_test.dart

```diff
@@ -97,5 +101,51 @@ void main() {
         expect(readme2.textContent, readme.textContent);
       },
     );
+
+    testWithProfile(
+      'Update pubspec in PackageVersion and in PackageVersionAsset',
+      fn: () async {
+        final pv =
+            (await packageBackend.lookupPackageVersion('oxygen', '1.2.0'))!;
+        final asset = (await packageBackend.lookupPackageVersionAsset(
+          'oxygen',
+          '1.2.0',
+          AssetKind.pubspec,
+        ))!;
+        final originalPvContent = pv.pubspec!.jsonString;
+        final originalAssetContent = asset.textContent;
+
+        pv.pubspec = Pubspec.fromYaml('name: x');
```

Comment:
Can we leave a comment with some explanation here, like:
```suggestion
        // Inject non-normalized version number in the pubspec asset.
        // This ensures that we can cleanup from:
        // https://github.com/dart-lang/pub_semver/pull/63
        pv.pubspec = Pubspec.fromYaml('name: x');
```